windows/ReactTestApp/MainPage.cpp

```diff
@@ -131,8 +121,23 @@ namespace winrt::ReactTestApp::implementation
         TitleBar().Height(sender.Height());
     }
 
-    void WritePropertyValue(std::any propertyValue,
-                            winrt::Microsoft::ReactNative::IJSValueWriter writer)
+    void MainPage::WriteInitialProperties(
+        std::optional<std::map<std::string, std::any>> const &initialProps)
+    {
+        if (initialProps.has_value()) {
+            ReactRootView().InitialProps([initialProps](IJSValueWriter const &writer) {
```

Comment:
Avoid copying `initialProps`:
```suggestion
            ReactRootView().InitialProps([&initialProps](IJSValueWriter const &writer) {
```